src/unix/apple/system.rs

```diff
@@ -531,6 +533,46 @@ impl SystemInner {
         physical_core_count()
     }
 
+    #[cfg(all(target_os = "macos", not(feature = "apple-sandbox")))]
+    pub(crate) fn motherboard_name() -> Option<String> {
```

Comment:
Instead of duplicating all methods, could you move the `#[cfg()]` checks inside their bodies please?

Reply:
By that I mean:

```rust
    pub(crate) fn motherboard_name() -> Option<String> {
        cfg_if!(
            if #[cfg(all(target_os = "macos", not(feature = "apple-sandbox")))] {
                get_io_platform_property("board-id")
            } else {
                None
            }
        )
    }
```